oci-unit-tests/helper/test_helper.sh

```diff
@@ -97,3 +100,61 @@ check_manifest_exists()
     debug "not found"
     return 1
 }
+
+# Check that the images listed
```

Comment:
Is there any typos in this comment? not sure if it is clear to me...

Reply:
Heh, not a typo per se, but I clearly forgot to finish the sentence.  Thanks.